### ISSUE TITLE
Scheduling / Reporting workflow not linked 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tamzi/bungeni-portal#434]
```
Currently there is no logical connection between the workflow of creating the 
schedule and creating the reports. 

This leads to logical defects :

-- it is possible to create votes and proceedings without having a agenda 

-- items can be set as “defered” on the agenda

The scheduling needs to work like this :

-- Initially on launch - scheduling screen is in draft mode.

-- Clerk drags and drops items on the scheduling screen 

--- The action drop-down should allow only preview of the agenda (and should 
not show "votes and proceedings" )

-- Items on the schedule whose states can be changed - should have only 
"scheduled" or in the case of the bill ... "bills scheduled for first 
reading"  Only states related to the item being scheduled must be allowed 
(e.g. "deferred" is a state that should not be allowed, since the item is 
"deferred" only after the sitting has occured.

-- Once the schedule has been set from "draft" to "published" the system must 
allow generation of an agenda. Perhaps an "agenda published" state is 
required to indicate that the final agenda has been published.

-- Only when the schedule is in "agenda published" state, and after the 
sitting has taken place -- the "Votes and Proceedings" option must be 
provided on the actions menu.

-- Perhaps a "draft votes and proceedings" state is required to initiate the 
"votes and proceedings" creation.

-- When the schedule is in "draft votes and proceedings" state (or when a 
similar state has been reached) the items on the schedule will only have the 
individual item actions for post-operative states (e.g. "deferred") 

-- The clerk can reorder the items on the schedule (this should not affect 
the order of the items as they were displayed on the agenda ?? ).

-- Once the clerk is satisfied - he clicks the "publish votes and 
proceedings" action to publish the votes and proceedings
```

Original issue reported on code.google.com by `ashok.ha...@gmail.com` on 19 Oct 2009 at 8:29
